modules/openapi-generator/src/main/resources/cpp-qt5-client/api-body.mustache

```diff
@@ -123,6 +104,49 @@ void {{classname}}::setNetworkAccessManager(QNetworkAccessManager* manager) {
     _manager = manager;  
 }
 
+    /**
+     * Appends a new ServerConfiguration to the config map for a specific operation.
+     * @param operation The id to the target operation.
+     * @param URL A string that contains the URL of the server
+     * @param description A String that describes the server
+     * @param variables A map between a variable name and its value. The value is used for substitution in the server's URL template.
+     * returns the index of the new server config on success and -1 if the operation is not found
+     */
+int {{classname}}::addServerConfiguration(const QString &operation, const QString &URL, const QString &description, const QMap<QString, {{prefix}}ServerVariable> &variables){
```

Comment:
```suggestion
int {{classname}}::addServerConfiguration(const QString &operation, const QUrl &url, const QString &description, const QMap<QString, {{prefix}}ServerVariable> &variables){
```

Reply:
What about using `QUrl` here?
I also used minor case for the parameter name.

Reply:
I think the `setServerIndex` accepts a string, but I am open if using one or the other is more intuitive.

Reply:
The reason for `QString` here is that a server like this `{host}:{port}` will be parse with regular expressions to fill the fields later on. Since `Qurl` has no replace function I decided to just use a `QString`. If that needs to be changed, the class `ServerConfiguration` needs to be changed aswell

Reply:
You are exposing your implementation details to the user of the client. I do not feel too strongly about it, but a QUrl makes sense to me for the function signature, and you can .toString() if you want in the function to accomodate your logic/implementation needs. Just a thought.

Reply:
OK for me either way.

Reply:
As I said, this would require changes on other classes aswell. Im fine either way too but I cant provide the `Qurl` change this week. Maybe next week if its needed!

Reply:
If the first line of the function converts the qurl parameter to a string, no other changes are necessary....

Reply:
Changed to QUrl. See latest commit